src/datachain/lib/dc.py

```diff
@@ -577,6 +572,43 @@ def datasets(
             **{object_name: datasets},  # type: ignore[arg-type]
         )
 
+    @classmethod
+    def listings(
+        cls,
+        session: Optional[Session] = None,
+        in_memory: bool = False,
+        object_name: str = "listing",
+        **kwargs,
+    ) -> "DataChain":
+        """Generate chain with list of cached listing.
```

Comment:
listing -> listings 

Q: do we really need to make it a public dc method? (I don't have any strong opinion on this - just thinking if we can for now expose less and see if there are scenarios for this)

Reply:
does it mean we will need `from_listing` btw? (just trying to think if we can keep at all about datasets / datachain and avoid exposing "listing" concept to end users (?)

Reply:
Yea, we do have usage of both `listings` and `storages` words around interface which is ok IMO. Listing is just listed storage so I don't think those two words cannot live together. 
I don't think we need `from_listing` as we have `from_storage` which in background creates new listing if doesn't exist and uses it then.

I think it's ok for us to expose this as we will be using it in Studio for example which can also be seen as one client.